src/genome.c

```diff
@@ -91,6 +127,11 @@ int genome_mutate_insertion_deletion(sc_population * population, sc_genome * ind
 	if (mutation_type == 1) {
 		/* add an upgrade step */
 		if (individual->steps < population->sys.no_of_programs-1) {
+			/* create another installation step */
+			if (genome_create_installation_step(population,
+												individual,
```

Comment:
Hopefully just a tabspace issue with this UI. What are you using for tab width? (assumes 4) Not overly important, just keeps things consistent for viewing